src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -28,6 +29,9 @@ export const monoPlatform: Platform = {
     });
   },
 
+  invokeDotNetMethod : invokeDotNetMethod,
+  invokeDotNetMethodAsync: invokeDotNetMethodAsync,
```

Comment:
The layering here doesn't seem right.

`invokeDotNetMethod` and `invokeDotNetMethodAsync` are implemented by calling platform's `callMethod`, which looks good. But platform itself doesn't need to expose `invokeDotNetMethod` or `invokeDotNetMethodAsync`.

Can we remove `invokeDotNetMethod`/`invokeDotNetMethodAsync` from the `Platform` contract? Anyone implementing `Platform` only needs to supply implementations of the low-level `findMethod`/`callMethod`, and then the higher-level `invokeDotNetMethod`/`invokeDotNetMethodAsync` will automatically work on top of them.

Reply:
@SteveSandersonMS Do you feel this should just hang of the Blazor object?

Reply:
Yes